engine/apps/user_management/sync.py

```diff
@@ -12,18 +12,29 @@
 
 
 def sync_organization(organization):
-    client = GrafanaAPIClient(api_url=organization.grafana_url, api_token=organization.api_token)
+    grafana_api_client = GrafanaAPIClient(api_url=organization.grafana_url, api_token=organization.api_token)
```

Comment:
super nit: `gcom_client` is set inside the conditional, so `grafana_api_client` could set there as well for consistency :)